src/_guides/language/language-tour.md

```diff
@@ -3713,6 +3586,16 @@ Future checkVersion() [!async!] {
 }
 {% endprettify %}
 
+<aside class="alert alert-info" markdown="1">
+**Note:**
+Although an async function might perform time-consuming operations,
+it doesn't wait for those operations.
+Instead, the async function executes only until it encounters
+its first `await` expression.
```

Comment:
Good note. We should be able to refer the reader to a page with more details. How about using the link to https://github.com/dart-lang/sdk/blob/master/docs/newsletter/20170915.md#synchronous-async-start for now?